### PR TITLE
updated font, logo, wordmark, and colors

### PR DESCRIPTION
**Pull request by jodymlin**
* Updated font and font family
  * Display/Header font: Poppins
  * Body: Open Sans
* Updated new colors where applicable (not all of our palette has new colors, but we also never use some of our palette anyway)
  * specifically: palette.secondary.light and palette.secondary.dark did not get updated
* switched to new logo and wordmark
* Componentized the page headers 
* Change some typography variants on some pages to look nicer with the new font family update

**Note**: blocked for merging until 10/4/2020

### CODE REVIEW

File: src/components/HeadFooter/HeadFooter.js [hunk 1]
```diff
@@ -11,29 +11,29 @@ const overwrittenTheme = createMuiTheme({
...
+			light: '#9C28C5',
+			dark: '#3E2961',
 			contrastText: '#fff'
 		},
 		secondary: {
-			main: '#ED3266',
-			light: '#f05b84',
-			dark: '#a52347',
+			main: '#FF477E',
+			light: '#f05b84', // did not receive updates for this color
```
Comment (TimothyGu) [inline src/components/HeadFooter/HeadFooter.js:20]: The comment won't make sense after a year. To convey some information to the reviewer, you can instead comment on your own PRs.

File: src/components/PageTitle/PageTitle.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	return <Typography variant="h2" component='h1' className={classNames(classes.headline, className)} {...props} />;
+}
+
+PageTitle.propTypes = {
+	className: PropTypes.string
+};
+
+export default PageTitle;
+
+
```
Comment (TimothyGu) [inline src/components/PageTitle/PageTitle.js:25]: remove the empty new lines please. surprised ESLint didn't catch this.

**Review by khxia — COMMENTED** [review]
There's some empty lines here

**Review by GalenWong — CHANGES_REQUESTED** [review]
- Change the blog post cards on the homepage to use OpenSans
- The title weight on the markdown pages (blogs) is now invalid. We should either import opensans with those weights or we update our weight number, or we should just use poppins
- maybe we should use 600/700 for title poppins font weights
- remove `font-weight: lighter` on team page officer descriptions

File: src/components/HeadFooter/HeadFooter.js [hunk 3]
```diff
@@ -26,14 +26,14 @@ const overwrittenTheme = createMuiTheme({
...
-		h1: { fontFamily: '"Palanquin Dark", sans-serif' },
-		h2: { fontFamily: '"Palanquin Dark", sans-serif' },
-		h3: { fontFamily: '"Palanquin Dark", sans-serif' },
-		h4: { fontFamily: '"Palanquin Dark", sans-serif' },
-		h5: { fontFamily: '"Palanquin Dark", sans-serif' },
-		h6: { fontFamily: '"Palanquin Dark", sans-serif' },
-		body1: { fontFamily: '"Chivo", sans-serif' },
-		button: { fontFamily: '"Chivo", sans-serif' }
+		h2: { fontFamily: '"Poppins", sans-serif', fontWeight: '600' },
+		h1: { fontFamily: '"Poppins", sans-serif', fontWeight: '600' },
```
Comment (GalenWong) [inline src/components/HeadFooter/HeadFooter.js:30]:
```suggestion
		h1: { fontFamily: '"Poppins", sans-serif', fontWeight: '600' },
		h2: { fontFamily: '"Poppins", sans-serif', fontWeight: '600' },
```

**Review by GalenWong — CHANGES_REQUESTED** [review]

**Review by GalenWong — APPROVED** [review]

**Pull request merged**